### DISCUSSION TITLE
[BUG] Get all runes not working - A PSA

### DISCUSSION THREAD

**Discussion by LeoBadeaux** [Prodigy-Hacking/ProdigyMathGameHacking#869] [(B) Bug Discussion]
We've received quite a few reports about the "Get all runes" feature not working. We are looking into this, and we urge you not to create any new issues about this.

**Comment by S0vereignity** [comment]
Ok
(ty because I was just about to report it)

**Reply by ghost** [reply]
*gasp* https://office365.discoveryeducation.com/learn/videos/f02aa83e-dc4a-48be-9482-bbabfa5b7a0e
